requirements.txt

```diff
@@ -0,0 +1,21 @@
+aiohttp==3.8.3 ; python_version >= "3.8" and python_version < "4.0"
```

Comment:
for my curiosity, what is the purpose of this file please?

Reply:
@TheJuanAndOnly99 @maoo please advise

Reply:
If you use poetry, the requirements.txt should not be needed. I believe that Juan pushed it by mistake in the PR as it was not listed in .gitignore . Feel free to remove it.

Reply:
just found it.  it is used by safety , https://github.com/pyupio/safety
@symphony-soufiane